src/table/components/head/TableHeadWrapper.tsx

```diff
@@ -69,6 +70,12 @@ function TableHeadWrapper({
             changeSortOrder(column, sortOrder);
           };
 
+          // Check context menu state.
+          // If it is open, clicking on the head cell to close the menu will not cause changing sort order.
+          const menuState = (state: boolean) => {
+            setMenuOpen(state);
+          };
```

Comment:
this is a pointless wrapper, might as well just use `setMenuOpen`

Reply:
Yes true, if we keep just menuOpen useState here then it makes no sense